docs/diagrams/ChainCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager"  as LogicManager LOGIC_COLOR
+participant ":ChainCommand"  as ChainCommand LOGIC_COLOR
+participant ":Command"  as Command LOGIC_COLOR
+participant ":CommandResult"  as CommandResult LOGIC_COLOR
+end box
+
+create ChainCommand
+LogicManager -> ChainCommand : execute()
+loop for all command in commands
+    activate ChainCommand
+    create Command
+    ChainCommand -> Command : execute()
+    create CommandResult
+    Command -> CommandResult : create
+    activate CommandResult
+    CommandResult -> CommandResult : getFeedbackToUser()
```

Comment:
Why would `CommandResult` call its own feedback method?

Reply:
is that a method from CommandResult to get its own attribute? or where shd it been called?